examples/Cargo.toml

```diff
@@ -16,6 +16,10 @@ libloading = { version = "0.7.0", optional = true }
 path = "../gtk4"
 package = "gtk4"
 
+[dependencies.gsk]
```

Comment:
is this needed? you can use gsk from gtk directly

Reply:
Indeed, fixed